e2e-test/tsps_e2e_test.py

```diff
@@ -35,21 +35,44 @@ def update_imputation_pipeline_workspace_id(tsps_url, workspace_id, token):
     logging.info(f"successfully updated imputation pipeline workspace id: {workspace_id}")
 
 
-# run imputation beagle pipeline
-def run_imputation_pipeline(tsps_url, token):
+def prepare_imputation_pipeline(tsps_url, token):
     request_body = {
-        "description": "string",
-        "jobControl": {
-            "id": f'{uuid.uuid4()}'
-        },
+        "jobId": f'{uuid.uuid4()}',
         "pipelineVersion": "string",
         "pipelineInputs": {
             "multiSampleVcf": "this/is/a/fake/file.vcf.gz",
             "outputBasename": "fake_basename"
         }
     }
 
-    uri = f"{tsps_url}/api/pipelineruns/v1/imputation_beagle"
+    uri = f"{tsps_url}/api/pipelineruns/v1/imputation_beagle/prepare"
+    headers = {
+        "Authorization": f"Bearer {token}",
+        "accept": "application/json",
+        "Content-Type": "application/json"
+    }
+
+    response = requests.post(uri, json=request_body, headers=headers)
+    status_code = response.status_code
+
+    if status_code != 200:
+        raise Exception(response.text)
+
+    logging.info(f"Successfully prepared imputation pipeline run")
+    response = json.loads(response.text)
+
+    return response['jobId'], response['pipelineFileInputs']
```

Comment:
hmm should we rename `pipelineFileInputs` to something else to make it more obvious that it contains sas urls that the user should use to upload with?